src/core/records-grant-authorization.ts

```diff
@@ -69,8 +69,21 @@ export class RecordsGrantAuthorization {
     } else if (grantScope.protocol !== recordsWrite.message.descriptor.protocol) {
       throw new DwnError(
         DwnErrorCode.RecordsGrantAuthorizationScopeProtocolMismatch,
-        `Grant scope specifies different protocol than in record with recordId ${recordsWrite.message.recordId}`
+        `Grant scope specifies different protocol than what appears in the record`
       );
+    } else {
```

Comment:
Optional style nit: do you think four flat `if` without `else` or nesting is nicer to the eye? Okay if you prefer to leave as is.

Reply:
Updated